modules/survey_accounts/php/addsurvey.class.inc

```diff
@@ -110,9 +110,16 @@ class AddSurvey extends \NDB_Form
         $reminder        = " already exists for given candidate for visit ";
         foreach ($instrument_list as $instrument) {
             if ($values['Test_name'] == $instrument['Test_name']) {
+                $test_name = $db->pselectRow(
```

Comment:
Should use `pselectOne`, this way, you'll have a single value instead of an array.
```suggestion
                $test_name = $db->pselectOne(
```